policy/modules/system/sysnetwork.te

```diff
@@ -280,6 +281,11 @@ optional_policy(`
 	xen_append_log(dhcpc_t)
 ')
 
+optional_policy(`
+	udev_search_runtime(dhcpc_t)
+	udev_read_runtime_files(dhcpc_t)
+')
```

Comment:
The `udev_search_runtime()` is a subset of `udev_read_runtime_files()`, so it isn't necessary.

Also this block should go above the userdom optional block, on line 269.